### ISSUE TITLE
systemd-journald repeats same message too much

### ISSUE THREAD

**Issue by Avamander** [systemd/systemd#14591]
**systemd version the issue has been seen with**
systemd 242

**Used distribution**
Ubuntu 19.10

**Expected behaviour you didn't see**
systemd-journald calms down with it's logging so it doesn't flood out anything relevant

**Unexpected behaviour you saw**
systemd-journald spammed the log lines, making it impossible to debug why a filesystem went read-only

**Steps to reproduce the problem**
Turn the root filesystem read-only after the system has booted

**Log**
```
[1758248.955796] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758248.955811] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758248.957821] systemd-journald[743]: Failed to write entry (27 items, 730 bytes) despite vacuuming, ignoring: Argument list too long
[1758248.957872] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758248.957880] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758248.959880] systemd-journald[743]: Failed to write entry (23 items, 794 bytes) despite vacuuming, ignoring: Argument list too long
[1758248.959957] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758248.959965] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758248.961533] systemd-journald[743]: Failed to write entry (23 items, 714 bytes) despite vacuuming, ignoring: Argument list too long
[1758248.961886] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758254.773714] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758254.773730] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758254.775874] systemd-journald[743]: Failed to write entry (24 items, 736 bytes) despite vacuuming, ignoring: Argument list too long
[1758254.775939] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758254.775948] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758254.777794] systemd-journald[743]: Failed to write entry (24 items, 736 bytes) despite vacuuming, ignoring: Argument list too long
[1758254.778245] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758254.778254] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758254.779933] systemd-journald[743]: Failed to write entry (22 items, 647 bytes) despite vacuuming, ignoring: Argument list too long
[1758254.780245] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758276.210480] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758276.210488] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758276.211694] systemd-journald[743]: Failed to write entry (22 items, 647 bytes) despite vacuuming, ignoring: Argument list too long
[1758276.211854] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758276.211859] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758276.212713] systemd-journald[743]: Failed to write entry (22 items, 753 bytes) despite vacuuming, ignoring: Argument list too long
[1758276.212851] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758276.212855] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758276.213697] systemd-journald[743]: Failed to write entry (22 items, 623 bytes) despite vacuuming, ignoring: Argument list too long
[1758276.213835] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758281.297293] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758281.297304] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758281.299357] systemd-journald[743]: Failed to write entry (24 items, 736 bytes) despite vacuuming, ignoring: Argument list too long
[1758281.299416] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758281.299424] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758281.301264] systemd-journald[743]: Failed to write entry (24 items, 736 bytes) despite vacuuming, ignoring: Argument list too long
[1758284.187483] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
[1758284.187494] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/user-1000.journal: Read-only file system
[1758284.189664] systemd-journald[743]: Failed to write entry (23 items, 694 bytes) despite vacuuming, ignoring: Argument list too long
[1758284.189709] systemd-journald[743]: Failed to rotate /var/log/journal/4682ff87681c483b92e83a97ee93503f/system.journal: Read-only file system
```

**Comment by poettering** [comment]
Duplicate of #2339
